linkify.js

```diff
@@ -15,10 +15,27 @@ function redirectGoLink(details) {
     return {redirectUrl: local_storage_cache[golink].url}
 }
 
+function redirectSearchLink(details) {
+    let golink = details.url.replace(/(.*)\:\/\/(.*)=go%2F/, "")
+    const regex = /%2F/g;
```

Comment:
what is `g`? please comment

Reply:
It's the global flag of RegExp. We don't actually need it so I've removed it.